test/AzureExtension/DevBox/DevBoxTestsSetup.cs

```diff
@@ -0,0 +1,123 @@
+                    ""uri"": ""https://72f9feed-86f1-41af-91ab-beefd011db47-devcenter-dc.westus3.devcenter.azure.com/projects/nonprod/users/28feedaeb-996b-434b-bb57-4969cbeef8f0/devboxes/two"",
+                    ""name"": ""Two"",
+                    ""projectName"": ""Nonprod"",
+                    ""poolName"": ""Some-Enhanced-Pool-In-Da-West"",
+                    ""hibernateSupport"": ""Disabled"",
+                    ""provisioningState"": ""Succeeded"",
+                    ""actionState"": ""Stopped"",
+                    ""powerState"": ""Deallocated"",
+                    ""uniqueId"": ""28feedaeb-996b-434b-bb57-4969cbeef8f0"",
+                    ""location"": ""westus2"",
+                    ""osType"": ""Windows"",
+                    ""user"": ""28feedaeb-996b-434b-bb57-4969cbeef8f0"",
+                    ""hardwareProfile"": {
+                    ""vCPUs"": 16,
+                    ""skuName"": ""some_sku_v2"",
+                    ""memoryGB"": 64
+                    },
+                    ""storageProfile"": {
+                    ""osDisk"": {
+                        ""diskSizeGB"": 2048
+                    }
+                    },
+                    ""imageReference"": {
+                    ""name"": ""1es-office-enhancedtools-baseImage"",
+                    ""version"": ""2023.0608.0"",
+                    ""operatingSystem"": ""Windows 11 Enterprise"",
+                    ""osBuildNumber"": ""22H2"",
+                    ""publishedDate"": ""2023-06-08T19:48:22.7178982+00:00""
+                    },
+                    ""createdTime"": ""2023-08-22T19:06:51.0750942+00:00"",
+                    ""localAdministrator"": ""Enabled""
+                }
+                ],
+            ""facets"": [],
+            ""resultTruncated"": ""false"",
+            ""rdpConnectionUrl"": ""https://sample.uri.com"",
+            ""webUrl"": ""https://sample.uri.com""
+        }";
+
+    private HttpClient mockHttpClient = new();
+
+    [TestInitialize]
+    public void TestInitialize()
+    {
+        TestOptions = TestHelpers.SetupTempTestOptions(TestContext!);
+
+        // Create a mock HttpMessageHandler
+        var handlerMock = new Mock<HttpMessageHandler>();
+
+        handlerMock.Protected()
+            .Setup<Task<HttpResponseMessage>>("SendAsync", ItExpr.IsAny<HttpRequestMessage>(), ItExpr.IsAny<CancellationToken>())
+            .ReturnsAsync(new HttpResponseMessage
+            {
+                StatusCode = HttpStatusCode.OK,
+                Content = new StringContent(MockTestJson),
+            });
+
+        // Create an HttpClient using the mocked handler
+        mockHttpClient = new HttpClient(handlerMock.Object);
+
+        mockFactory.Setup(f => f.CreateClient(It.IsAny<string>()))
+            .Returns(mockHttpClient);
+    }
+
+    [TestCleanup]
+    public void Cleanup()
+    {
+        TestHelpers.CleanupTempTestOptions(TestOptions, TestContext!);
+    }
+
+    public void Dispose()

```

Comment:
Doesn't seem like there is anything to dispose. You can remove IDisposable.

Reply:
Auto added by VS. Will remove.